packages/snap-preact-components/src/components/Molecules/FacetListOptions/readme.md

```diff
@@ -33,9 +33,9 @@ If using within Autocomplete, the `previewOnFocus` prop will invoke the `value.p
 
 ```jsx
 <Autocomplete>
-	...
-	<FacetGridOptions values={sizeFacet.values} previewOnFocus={true} />
-	...
+	
```

Comment:
I think these dots are useful to show that this example is not complete, but just a snippet.